src/uwtools/tests/drivers/test_make_solo_mosaic.py

```diff
@@ -0,0 +1,119 @@
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+"""
+make_solo_mosaic driver tests.
+"""
+from unittest.mock import DEFAULT as D
+from unittest.mock import patch
+
+import yaml
+from pytest import fixture
+
+from uwtools.drivers import make_solo_mosaic
+from uwtools.scheduler import Slurm
+
+# Fixtures
+
+
+@fixture
+def config(tmp_path):
+    return {
+        "make_solo_mosaic": {
+            "config": {
+                "dir": str(tmp_path / "input" / "input_grid_file"),
+                "num_tiles": 1,
+            },
+            "execution": {
+                "batchargs": {
+                    "cores": 1,
+                    "walltime": "00:01:00",
+                },
+                "executable": "/path/to/make_solo_mosaic.exe",
+            },
+            "run_dir": str(tmp_path),
+        },
+        "platform": {
+            "account": "me",
+            "scheduler": "slurm",
+        },
+    }
+
+
+@fixture
+def config_file(config, tmp_path):
+    path = tmp_path / "config.yaml"
+    with open(path, "w", encoding="utf-8") as f:
+        yaml.dump(config, f)
+    return path
+
+
+@fixture
+def driverobj(config_file):
+    return make_solo_mosaic.MakeSoloMosaic(config=config_file, batch=True)
+
+
+# Tests
+
+
+def test_MakeSoloMosaic(driverobj):
+    assert isinstance(driverobj, make_solo_mosaic.MakeSoloMosaic)
+
+
+def test_MakeSoloMosaic_dry_run(config_file):
+    with patch.object(make_solo_mosaic, "dryrun") as dryrun:
+        driverobj = make_solo_mosaic.MakeSoloMosaic(config=config_file, batch=True, dry_run=True)
+    assert driverobj._dry_run is True
+    dryrun.assert_called_once_with()
+
+
+def test_MakeSoloMosaic_provisioned_run_directory(driverobj):
+    with patch.multiple(
+        driverobj,
+        runscript=D,
+    ) as mocks:
+        driverobj.provisioned_run_directory()
+    for m in mocks:
+        mocks[m].assert_called_once_with()
```

Comment:
```suggestion
def test_MakeSoloMosaic_provisioned_run_directory(driverobj):
    with patch.object(driverobj, "runscript") as runscript:
        driverobj.provisioned_run_directory()
        runscript.assert_called_once_with()
```
Also, `patch.object()` is preferable, due to its comparative brevity, is preferable to `patch.multiple()` when only one item is being patched/mocked.